### ISSUE TITLE
Properly handle RSVP

### ISSUE THREAD

**Issue by wagenet** [embroider-build/embroider#1195]
https://github.com/ember-animation/ember-animated/issues/451

**Comment by NullVoxPopuli** [comment]
Can ember-animated use normal Promise?

**Comment by ef4** [comment]
Yes but the bug is a real bug. Switching away from rsvp is a workaround. 

The bug is that embroider needs to prioritize the copies of packages that ship inside ember-source over copies that happen to be floating around.

**Comment by NullVoxPopuli** [comment]
Related: https://github.com/emberjs/ember.js/pull/20349
